src/db/versions.ml

```diff
@@ -56,33 +56,54 @@ let init () =
       {|create table module_libraries(
         mdl_id int not null references module_index(mdl_id),
         mdl_lib_id int not null references library_index(lib_id),
-        mdl_lib varchar not null, 
+        mdl_lib varchar not null,
         primary key (mdl_id, mdl_lib_id)
       )|};
       {|create table module_vals(
         mdl_id int not null references module_index(mdl_id),
         mdl_name varchar not null,
         mdl_opam_name varchar not null references opam_index(opam_name),
         mdl_ident varchar not null,
-        mdl_val varchar not null, 
+        mdl_val varchar not null,
         primary key (mdl_id, mdl_ident)
-      )|}
-    ]
+      )|};
+      {|create table module_types(
+        mdl_id int not null references module_index(mdl_id),
+        mdl_opam_name varchar not null references opam_index(opam_name),
```

Comment:
You need to have mdl_name here as for **module_vals**. It will optimize future requests made by search-API.